### PR TITLE
[document] Add zh doucment  Typescript-sdk / account.mdx

### PR DESCRIPTION
**Pull request by yue1823**
### Description
 Add zh doucment Typescript-sdk / account.mdx
### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [ ] Have you ran `pnpm fmt`?
  - [ ] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/zh/build/sdks/ts-sdk/account.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+這裡有一些例子是教你怎麼使用具體的解碼方案：
+
+### 從私鑰生成帳戶
+
+SDK 支援使用「fromPrivateKey()」靜態方法從私鑰衍生出帳戶。
+另外，該方法支援從私鑰和帳戶地址衍生帳戶。
+此方法使用本地計算，因此適合用於衍生尚未輪換身份驗證金鑰的「帳戶」。
+
+```ts
+// to derive an account with a legacy Ed25519 key scheme
```
Comment (gregnazario) [inline apps/nextra/pages/zh/build/sdks/ts-sdk/account.mdx:55]: Should we translate these to Chinese too?

**Review by gregnazario — APPROVED** [review]

File: apps/nextra/pages/zh/build/sdks/ts-sdk/account.mdx (hunk 1 above)
Comment (yue1823) [reply apps/nextra/pages/zh/build/sdks/ts-sdk/account.mdx:55]: I am not sure the note typescript supports Chinese. But if you want, I can do so.

Comment (gregnazario) [reply apps/nextra/pages/zh/build/sdks/ts-sdk/account.mdx:55]: That's okay then, we can skip for now

**Comment by gregnazario** [comment]
Thanks for your contribution!  I really appreciate it!

**Pull request merged**